tests/testthat/test-measures.R

```diff
@@ -51,3 +51,16 @@ test_that("riem_measures checks dates",{
                "date_end has to be bigger than date_start")
 })
 
+test_that("riem_measures returns correct values when alternate arguments to missing are passed",{
+  skip_on_cran()
+  output <- riem_measures(station = "VOHY", date_start = "2014-01-01",
+                          date_end = "2016-04-22", missing = "M")
```

Comment:
So when you choose "M", R will consider the whole column to be character, right? So why would one do that, actually?